core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/clazz/detailoverview/ClazzDetailOverviewViewModel.kt

```diff
@@ -244,7 +251,7 @@ class ClazzDetailOverviewViewModel(
             ARG_CLAZZUID to entityUidArg.toString(),
         )
 
-        when(courseBlock.cbType) {
+        when (courseBlock.cbType) {
```

Comment:
Please do not change spacing or reformat when no other change is being made. It just makes the pull request harder to review.